blackbelt/handle_github.py

```diff
@@ -15,6 +15,14 @@
     comment_ticket,
     move_to_deployed
 )
+
+from .git import (
+    get_current_branch,
+    merge as git_merge,
+    get_github_repo,
+    get_remote_repo_info
+)
```

Comment:
this may be actually worth doing sth like `git.get_remote_repo_info()`, `git.merge()`, etc.